pdcupdater/commands.py

```diff
@@ -35,7 +35,7 @@ def retry():
 def _initialize_basics(pdc):
     """ Gotta have these before we can really do anything... """
     arches = [dict(name=name) for name in ["armhfp", "i386", "x86_64"]]
-    pdc_arches = list(pdc_client.get_paged(pdc['arches']._))
+    pdc_arches = list(pdc.get_paged(pdc['arches']._))
```

Comment:
The class has this method since pdc-client-0.9.0 (Dec 8, 2015, commit 85ec4e5b9dc088e3583e55c27dc13ccd166df370). Your call.